### PR TITLE
Fix show method after updating container

### PR DESCRIPTION
**Pull request by MasGaNo**

### CODE REVIEW

File: source/view.ts [hunk 1]
```diff
@@ -753,7 +753,14 @@ class View extends Backbone.View<Backbone.Model> {
...
 
-        $container.css('display', displayCss);
+        if (!displayCss) {
+            $container.each((index, elem: HTMLElement) => {
+                elem.style.display = ''; // Fix because jQuery doesn't remove inline style display: none if displayMode is null or undefined.
+            });
+        } else {
+            $container.css('display', displayCss);
+        }
+
```
Comment (FabianLauer) [inline source/view.ts]:
```
$container.css('display', displayCss || '');
```
Could this work instead? Might be more efficient.
See [this SO answer](https://stackoverflow.com/a/4036868).

File: source/view.ts [hunk 2]
```diff
@@ -810,8 +817,13 @@ class View extends Backbone.View<Backbone.Model> {
...
 
-        $container.css('display', displayCss);
-
+        if (!displayCss) {
+            $container.each((index, elem: HTMLElement) => {
+                elem.style.display = ''; // Fix because jQuery doesn't remove inline style display: none if displayMode is null or undefined.
+            });
+        } else {
+            $container.css('display', displayCss);
+        }
```
Comment (FabianLauer) [inline source/view.ts]: same here

File: source/view.ts [hunk 3]
```diff
@@ -833,19 +845,23 @@ class View extends Backbone.View<Backbone.Model> {
 
         }
 
-        this.$el.css('display', displayMode);
+        if (!displayMode) {
+            this.$el[0].style.display = ''; // Fix because jQuery doesn't remove inline style display: none if displayMode is null or undefined.
+        } else {
+            this.$el.css('display', displayMode);
+        }
```
Comment (FabianLauer) [inline source/view.ts]: same here too

File: source/view.ts (hunk 1 above)
Comment (MasGaNo) [reply source/view.ts]: Ouch... First it doesn't work, maybe I mistype something, good catch 👍

**Pull request merged**